wordpress-mobile/WordPress-iOS
# Add organizationID to ReaderTeamTopic

Ref: https://github.com/wordpress-mobile/WordPress-iOS/issues/15343, https://github.com/wordpress-mobile/WordPress-iOS/pull/15484#discussion_r542790416
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/316

This adds `organizationID` to `ReaderTeamTopic`. That is now used to determine the `organizationType` (instead of the `slug`).

To test:
There should be no visible changes from https://github.com/wordpress-mobile/WordPress-iOS/pull/15484 so smoke testing the site filters should be sufficient.

- Go to Reader > P2s.
  - Select `Filter`.
  - Verify the `Sites` list only contains non-A8C P2s. 
- Go to Reader > Automattic.
  - Select `Filter`.
  - Verify the `Sites` list only contains A8C P2s. 

PR submission checklist:
